doc/release-notes.rst

```diff
@@ -2,6 +2,142 @@
  Release Notes
 ===============
 
+v10.2.4 Jewel
+=============
+
+This point release fixes several important bugs in RBD mirroring, RGW multi-site, CephFS, and RADOS.
+
+We recommend that all v10.2.x users upgrade. Also note the following when upgrading from hammer
+
+Upgrading from hammer
+---------------------
+
+Set require_jewel_osds osdmap flag on upgrade from hammer
+
+When the last hammer OSD in a cluster containing jewel MONs is
+upgraded to jewel, as of 10.2.4 the jewel MONs will issue this
+warning: "all OSDs are running jewel or later but the
+'require_jewel_osds' osdmap flag is not set" and change the
+cluster health status to HEALTH_WARN.
+
+This is a signal for the admin to do "ceph osd set
+require_jewel_osds" - by doing this, the admin acknowledges that
+there is no downgrade path.
```

Comment:
that the upgrade is complete and no pre-jewel OSDs may be added to the cluster.  

I don't want to imply that there otherwise would be a downgrade path.. there isn't (a tested) one.